src/microbe_stage/PlayerMicrobeInput.cs

```diff
@@ -123,7 +123,7 @@ public override void _Process(float delta)
 
         if (toggleEngulf.ReadTrigger())
         {
-            if (stage.Player != null)
+            if (stage.Player != null && !stage.Player.Membrane.Type.CellWall)
```

Comment:
I think that the check for CellWall should be put in Microbe where engulfing is attempted to be turned on. I made the is engulfing a property just for this kind of case to be able to block going to engulfing mode in some cases.